airflow-core/tests/unit/models/test_dagrun.py

```diff
@@ -1511,9 +1511,6 @@ def task_2(arg2): ...
         (1, TaskInstanceState.REMOVED),
         (2, TaskInstanceState.REMOVED),
     ]
-    assert (
-        f"Removing task '{ti_2}' as the map_index is longer than the resolved mapping list (0)" in caplog.text
-    )
```

Comment:
There are some issues with loggers at the moment. `Structlog` and `stdlib` mixed together.

Removing for now as the assert above checking the that the TI state is REMOVE is enough.